### PR TITLE
254838: Updates to infra helm chart and appConfig

### PR DESCRIPTION
**Pull request by jeevankuduvaravindran**
Updates to infra helm chart and appConfig

[AB#254838](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/254838)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-web",
   "description": "Digital service mock to claim public money in the event property subsides into mine shaft.",
-  "version": "4.32.4",
+  "version": "4.32.7",
```
Comment (kbabigumira) [inline package.json:4]: shouldn't this be 4.32.5?

File: appConfig/appConfig.snd1.json [hunk 2]
```diff
@@ -97,6 +97,13 @@
             "label": "{{serviceName}}",
             "content_type": "{{keyVaultContentType}}",
             "tags": {}
+        },
+        {
+            "key": "Sentinel",
```
Comment (kbabigumira) [inline appConfig/appConfig.snd1.json:102]: Is this a shared key?

File: package.json (hunk 1 above)
Comment (jeevankuduvaravindran) [reply package.json]: @kbabigumira done

File: appConfig/appConfig.snd1.json (hunk 2 above)
Comment (jeevankuduvaravindran) [reply appConfig/appConfig.snd1.json:102]: @kbabigumira It is separate for each application differentiated by label

**Review by DaRoszko — APPROVED** [review]

**Review by ganeshg2491 — APPROVED** [review]

File: helm/ffc-demo-web/values.yaml [hunk 3]
```diff
@@ -86,7 +84,5 @@ containerConfigMap:
       secretName: ffc-demo-web-container-secret
       refresh:
         interval: 2m
-
-managedIdPrefix: sndadpinfmid1401
 userAssignedIdentity:
-  managedIdName: ffc-demo-services
\ No newline at end of file
+  managedIdName:
```
Comment (ganeshg2491) [inline helm/ffc-demo-web/values.yaml:88]: userAssignedIdentity can be removed.

**Review by ganeshg2491 — APPROVED** [review]

**Review by ajaysoni-cap — APPROVED** [review]

**Review by ppotturi — APPROVED** [review]

**Pull request merged**